core/src/utils/animation/test/animationbuilder/animation.e2e.ts

```diff
@@ -0,0 +1,43 @@
+import type { E2EPage } from '@utils/test/playwright';
+import { test } from '@utils/test/playwright';
+
+test.describe('animation: animationbuilder', async () => {
+  test.beforeEach(({ skip }) => {
+    skip.rtl();
+  });
+  test('backwards-compatibility animation', async ({ page }) => {
+    await page.goto('/src/utils/animation/test/animationbuilder');
+    await testNavigation(page);
+  });
+
+  test('ios-transition web', async ({ page, skip }) => {
+    skip.mode('md');
+
+    await page.goto('/src/utils/animation/test/animationbuilder');
+    await testNavigation(page);
+  });
+
+  test('ios-transition css', async ({ page, skip }) => {
+    skip.mode('md');
+
+    await page.goto('/src/utils/animation/test/animationbuilder?ionic:_forceCSSAnimations=true');
+    await testNavigation(page);
+  });
+});
+
+const testNavigation = async (page: E2EPage) => {
```

Comment:
Did you intentionally leave the screenshots off here? These tests seem redundant without them; we're basically just checking routing functionality at this point.

Reply:
This is supposed to check that the animations resolve when used in Ionic's transition system (for doing page transitions). If the animation does not resolve then `ionRouteDidChange` won't fire -- I can leave this out if you think this is already covered in `ion-router` though

Reply:
Huh, looking around it seems this is the only test checking the `ionRouteDidChange` event at the moment. Between that and the explicit `forceCSSAnimations` check, I suppose it couldn't hurt to keep.